view/rooms/app.js

```diff
@@ -27,7 +27,8 @@ const u = require('../utils');
 const uPage = require('./utils/page');
 const ROOM_ID = window.location.pathname.split('/rooms/')[ONE];
 const GET_BOTS = '/v1/bots';
-const GET_ROOM = '/v1/rooms/' + ROOM_ID;
+let GET_ROOM = '/v1/rooms/';
+GET_ROOM += isNaN(ROOM_ID) ? `?name=${ROOM_ID}` : ROOM_ID;
```

Comment:
Make sure that in your rooms controller that if the user is specifying a name with no wildcards, that we're calling .findOne on the db model rather than the more expensive .findAll/.findAndCountAll!